rules/windows/execution_via_explorer_suspicious_child_parent_args.toml

```diff
@@ -36,14 +36,18 @@ name = "Scripting"
 reference = "https://attack.mitre.org/techniques/T1064/"
 
 [[rule.threat.technique]]
-id = "T1192"
+id = "T1566"
+name = "Phishing"
+reference = "https://attack.mitre.org/techniques/T1566/"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.002"
 name = "Spearphishing Link"
-reference = "https://attack.mitre.org/techniques/T1192/"
+reference = "https://attack.mitre.org/techniques/T1566/002/"
 
-[[rule.threat.technique]]
-id = "T1193"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.001"
 name = "Spearphishing Attachment"
-reference = "https://attack.mitre.org/techniques/T1193/"
+reference = "https://attack.mitre.org/techniques/T1566/001/"
 
 
 [rule.threat.tactic]
```

Comment:
should be initial access

Reply:
https://github.com/brokensound77/detection-rules/blob/update-rule-threat-mappings/rules/windows/initial_access_via_explorer_suspicious_child_parent_args.toml